src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -179,6 +188,54 @@ The `terraform show` command shows detailed information of your cluster resource
 terraform show
 ~~~
 
+## Change a cluster's plan
+
+To change a CockroachDB {{ site.data.products.basic }} cluster's plan to CockroachDB {{ site.data.products.standard }} in place, or to change a CockroachDB {{ site.data.products.standard }} cluster to CockroachDB {{ site.data.products.basic }}, you can use Terraform.
+
+{{site.data.alerts.callout_info}}
+To migrate between CockroachDB {{ site.data.products.advanced }} and either CockroachDB {{ site.data.products.standard }} or CockroachDB {{ site.data.products.basic }}, you must create and configure a new cluster, back up the existing cluster's data, and restore the backup to the new cluster. Migration in place is not supported. Refer to [Customer-owned backups]({% link cockroachcloud/backup-and-restore-overview.md %}#customer-owned-backups).
+{{site.data.alerts.end}}
+
+To migrate from CockroachDB {{ site.data.products.basic }} to CockroachDB {{ site.data.products.standard }} in place:
+
+1. Edit the cluster's Terraform template:
+    - Change `plan` to `STANDARD`.
+    - Replace the contents of `serverless {}` (which may be empty) with the provisioned vCPUs for the cluster. This field is required for CockroachDB {{ site.data.products.standard }}. It is not possible to set storage limitations on CockroachDB {{ site.data.products.standard }}.
+      {% include_cached copy-clipboard.html %}
+      ~~~ hcl
+            serverless = {
+          usage_limits = {
+            provisioned_virtual_cpus = 2
+          }
+        }
+      ~~~
+1. Apply the template:
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    terraform apply
+    ~~~
+
+To migrate from CockroachDB {{ site.data.products.standard }} to CockroachDB {{ site.data.products.basic }} in place:
+
+1. Edit the cluster's Terraform template:
+    -  Change `plan` to `BASIC`.
```

Comment:
@mdlinville would it work to mention here that downgrading to Basic successfully requires that all Standard-only features be deactivated - e.g. any private endpoints must be deleted.